test/browser/clubs-page.test.jsx

```diff
@@ -97,6 +97,7 @@ describe("ClubsPage.ClubList", function() {
     owner: 'foo',
     website: 'http://example.org',
     location: 'Somewhere, USA',
+    status: 'approved',
     name: 'foo club'
   }];
 
```

Comment:
urghhhh stupid me, my [question above](https://github.com/mozilla/teach.webmaker.org/pull/965/files#r31479842) is still valid.  sorry!